src/backend/utils/github-url.js

```diff
@@ -1,7 +1,9 @@
 require('../lib/config');
 const parseGithubUrl = require('parse-github-url');
 const fetch = require('node-fetch');
+const { logger } = require('./logger');
 
+const log = logger.child({ module: 'github-url' });
```

Comment:
Let's not do this.  The docs suggest it, but for our needs, it's not really necessary to isolate logs into sub-modules at this stage.  Just use `logger.info()`, etc.